docs/components/IDGeneration.md

```diff
@@ -22,6 +22,7 @@ The two included in the package are:
 
```

Comment:
Shouldn't it say
> The _three_ included in the package are


Also, I wonder if one could clarify what is enumerated here, since all these are mentioned above: strategies, additional arguments, or requirements. (<-- this is me being very pedantic)